articles/architecture-scenarios/b2c/user-provisioning.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Consumer Identity and Access Management User Provisioning
+description: Understand how user provisioning works in your CIAM implementation. 
+toc: true
+topics:
+    - b2c
+    - ciam
+    - user-migration
+    - identity-store
+contentType: concept
+useCase:
+  - user-provisioning
+---
+# Consumer Identity and Access Management User Provisioning
+
+User provisioning defines how users will be added to your system. This part of the implementation determines where you will store your users' profile information, how you will migrate existing users, and how your users will sign up. 
+
+## Prerequisites
+
+* Tenant Architecture
+
+## Design considerations
+
+It is important to decide early in your implementation planning how you want users added. The decisions you make influence many other implementation choices.
+
+The factors to consider are: 
+
+1. Where to store your user data, in your own identity data store or in Auth's data store?
+2. If you have legacy user data, do you want to migrate it to Auth0?
+3. Do you want your users to sign up with their existing accounts like Facebook or Google? 
+
+## Identity data storage
+
+During sign up, Auth0 creates a [user profile](/users/concepts/overview-user-profile) that contains information about the user’s account. You can choose to store the user information in Auth0's data store or your own. 
+
+### Use your own identity data store
+
+If you want to use your own identity store, perhaps because you’ve got applications which you aren’t ready to migrate or can’t be migrated, use Auth0’s [Database Connections](/connections/database/custom-db) to proxy an existing identity store. 
```

Comment:
> Using your own data store is an option if you have users associated with applications that you aren't ready to migrate (or cannot be migrated). To do this, use Auth0's Database Connections functionality to proxy an existing identity store.

I also think the last part needs some rewriting, because I'm not sure people will know what "proxy an existing identity store" means?

Reply:
Yeah - I kinda hate that part too.